terraform/playground/instance.tf

```diff
@@ -5,6 +5,13 @@ resource "aws_eip" "playground" {
   }
 }
 
+resource "aws_eip" "playground2" {
+  domain = "vpc"
```

Comment:
aws [recommends](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/eip) using `domain` instead of `vpc = true`.

screenshot:


<img width="470" alt="image" src="https://github.com/user-attachments/assets/832a832e-4ed9-446b-a7af-6681980d54be">